### ISSUE TITLE
Cannot upload files if the directory has NO file/directory.

### ISSUE THREAD

**Issue by cellolism** [coderaiser/cloudcmd#145]
* **Version** (`cloudcmd -v`): v8.2.1
* **Node Version** `node -v`: v9.5.0
* **OS** (`uname -a` on Linux): Linux raspberrypi 4.9.59-v7+ #1047 SMP Sun Oct 29 12:19:23 GMT 2017 armv7l GNU/Linux
* **Browser name/version**: Chrome 63.0.3239.132
* **Used Command Line Parameters**: -a -u user -p password --root /home/pi/Videos/ --name app --prefix prefix --no-console
* **Changed Config**: -

If NO file/directory exists in the directory specified by `--root` option, I cannot upload any file to the directory and console say:
> Uncaught TypeError: Cannot read property 'replace' of undefined
>    at Object.t.exports [as uploadDirectory] (directory.js:23)
>    at HTMLElement.o (index.js:421)

kind regards,

**Comment by coderaiser** [comment]
If no `root` exist you can't event start `cloudcmd`:

```
coderaiser@cloudcmd:~$ cloudcmd --root /hello/world
url: http://localhost:31337/
cloudcmd --root: ENOENT: no such file or directory, stat '/hello/world'
```

**Comment by coderaiser** [comment]
I understood, you are talking about file upload with help of `drag'n'drop` when root directory is empty.

**Comment by coderaiser** [comment]
Thank you :), fixed with 90b8366. Landed in [v8.2.2](https://github.com/coderaiser/cloudcmd/releases/tag/v8.2.2).
